### PR TITLE
Prepare release 0.1.0

### PR DESCRIPTION
**Pull request by ckadner**
**Changes:**
- updated the version from `0.0.1` to `0.1.0`
- updated `description` and `long_description`

---

**Possible things to improve **_before_** we push to PyPi:**

Generate the `long_description` from the `sdk/README.md`. I will give that a try:

```Python
import setuptools

with open("sdk/README.md", "r") as f:
    long_description = f.read()
    # TODO: replace relative links with absolute links

setuptools.setup(
    name="...",
    description="...",
    long_description=long_description,
    long_description_content_type="text/markdown",
    # ...
)
```

### CODE REVIEW

**Comment by ckadner** [comment]
Btw, the failing tests are fixed in PR #174. I will rebase this PR once #174 is merged.

File: sdk/python/kfp_tekton/__init__.py [hunk 1]
```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.0.1'
+__version__ = '0.1'
```
Comment (Tomcli) [inline sdk/python/kfp_tekton/__init__.py:15]: nit:
```suggestion
__version__ = '0.1.0'
```

Comment (ckadner) [reply sdk/python/kfp_tekton/__init__.py]: Good point. Thanks @Tomcli

**Pull request merged**